### DISCUSSION TITLE
页面元数据是否支持 私有仓库地址？

### DISCUSSION THREAD

**Discussion by zuihou** [vuepress-theme-hope/vuepress-theme-hope#2954] [Q&A] [answered]
已经阅读如下文档：
https://vuepress-theme-hope.gitee.io/v2/zh/guide/feature/meta.html

在配置下图所示的功能时，是否支持 文档地址（docsRepo）是 私有仓库地址 的情况？
![image](https://user-images.githubusercontent.com/8500410/227720606-c6bc6d67-f097-43b1-9483-7edbb3cc3bff.png)

**Comment by Mister-Hope** [comment, marked as answer]
那这个按钮对别人来说有什么意义？
